SEED-platform/seed
# 3209 feat/export dataloggers and sensors

#### Any background context you want to provide?
This PR allows users to download the sensor and sensor reading data as a csv

#### How should this be manually tested?
Import sensor and sensor readings for a property using the following test files: 
[small_sensor_readings_example.csv](https://github.com/SEED-platform/seed/files/8532594/small_sensor_readings_example.csv)
[example-sensor-metadata.xlsx](https://github.com/SEED-platform/seed/files/8532595/example-sensor-metadata.xlsx)

Inventory > property (i) > sensors > create datalogger > upload sensors using "example-sensor-metadata.xlsx" > upload sensor readings using "small_sensor_readings_example.csv" > export as CSV for both sensors and readings

NOTE: see updated testing instructions in the comment below

#### What are the relevant tickets?
#3209 

#### Screenshots (if appropriate)

Sample generated CSVs
[Sensor_Usage_for_Property_3207429_2022_04_26.csv](https://github.com/SEED-platform/seed/files/8564953/Sensor_Usage_for_Property_3207429_2022_04_26.csv)
[Sensors_for_Property_3207429_2022_04_26.csv](https://github.com/SEED-platform/seed/files/8564954/Sensors_for_Property_3207429_2022_04_26.csv)
